src/swap/components/Swap.tsx

```diff
@@ -13,7 +13,9 @@ export function Swap({
   children,
   className,
   experimental = { useAggregator: true },
+  onError,
```

Comment:
This will make the experience between Swap and Transaction similar.

In the next couple PRs, than I need to make the breaking change of removing those from the Button component.